apps.awesim.org/apps/myjobs/initializers/ood_appkit_pitzer_exp_fix.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do |config|
   ActiveSupport.on_load(:active_record) do
     Workflow.where(batch_host: 'pitzer-exp').update_all(batch_host: 'pitzer')
-  end
+  end if File.exist?("#{ENV['HOME']}/ondemand/data/sys/myjobs/production.sqlite3")
```

Comment:
```
end if File.exist?(Configuration.production_database_path)
```

Reply:
actually `end if Configuration.production_database_path.file?`

Reply:
I can try in dev, but I didn't use it initially because I'm not sure if it exists at that time. The error happens in the `db:setup` rake task in `bin/setup-production`.

Reply:
Yep, that works! Change committed.